packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -35,8 +35,10 @@ export class AuthenticatorService implements OnDestroy {
     const authService = interpret(machine).start();
 
     this._machineSubscription = authService.subscribe((state) => {
-      this._authState = state;
-      this._facade = getServiceContextFacade(state);
+      this._authState = state as unknown as AuthMachineState;
```

Comment:
This was throwing a TS error, so I fixed the type

Reply:
Rather than cast twice (here and on line 40), can we just type the `state` argument on line 37?

Reply:
I can clean it up a little, by setting state to `unknown` then only typecasing to `AuthMachineState`. That helps clean it up a bit. But I'm not sure how to do that all in one in line 37. 


```

    this._machineSubscription = authService.subscribe((state: unknown) => {
      this._authState = state as AuthMachineState;
      this._facade = getServiceContextFacade(state as AuthMachineState);
    });
```